tests/test_fastavro.py

```diff
@@ -1723,3 +1723,53 @@ def test_appending_records_different_schema_fails(tmpdir):
             fastavro.writer(new_file, different_schema, [{"field": 1}])
 
         assert "does not match file writer_schema" in str(exc)
+
+
+def test_appendable_raises_valuerror(tmpdir):
```

Comment:
Do you mind moving these new tests into a new `test_six.py` file under the tests directory?